utils/utils.go

```diff
@@ -35,3 +36,13 @@ func WithRetries(times int, f func() error) error {
 	}
 	return err
 }
+
+//returns default Value if OS Enviroment Variable is not set
```

Comment:
comment should be of Form `//GetStringFromEnvWithDefault ...` -> `//GetStringFromEnvWithDefault returns default Value if OS Enviroment Variable is not set`